core/embed/firmware/header.S

```diff
@@ -26,8 +26,8 @@ g_header:
     . = . + 8                        // reserved
     . = . + 512                      // hash1 ... hash16
 
-#if defined TREZOR_MODEL_T
-// model T header specifics
+#if defined TREZOR_MODEL_T || defined TREZOR_MODEL_R
```

Comment:
let's flip this, because the comments aren't strictly correct
```
#if !defined TREZOR_MODEL_1
// trezor-core header style
...
#else
// model 1 compatibility header
```

Reply:
please do that in a separate commit because it is not emulator related

Reply:
So in the end all the changes in `header.S`, `startup.S`, `SConscript.firmware` etc. - not touching emulator - should end up in a separate commit?

Reply:
Done in [91b1938](https://github.com/trezor/trezor-firmware/pull/2230/commits/91b1938c506c2ea02489c5c4ee561eab3cbe1f4b) - during squashing, I would transfer all non-emulator changes into this commit